src/frontend/test_runner/tests/testdata/struct_query.yaml

```diff
@@ -32,287 +32,287 @@
   create_source:
     row_format: protobuf
     name: s
     file: |
-      syntax = "proto3";
-      package test;
-      message TestRecord {
-        int32 id = 1;
-        Country country = 3;
-        int64 zipcode = 4;
-        float rate = 5;
-      }
-      message Country {
-        string address = 1;
-        City city = 2;
-        string zipcode = 3;
-      }
-      message City {
-        int64 address = 1;
-        string zipcode = 2;
-      }
+        syntax = "proto3";
+        package test;
+        message TestRecord {
+          int32 id = 1;
+          Country country = 3;
+          int64 zipcode = 4;
+          float rate = 5;
+        }
+        message Country {
+          string address = 1;
+          City city = 2;
+          string zipcode = 3;
+        }
+        message City {
+          int64 address = 1;
+          string zipcode = 2;
+        }
+- sql: |
+    create materialized view t as select * from s;
+    select count(1), count((country).city.zipcode) from t where (country).city.address>1;
   logical_plan: |
     LogicalProject { exprs: [$0, $1], expr_alias: [ ,  ] }
       LogicalAgg { group_keys: [], agg_calls: [count($0), count($1)] }
         LogicalProject { exprs: [1:Int32, Field(Field($1, 1:Int32), 1:Int32)], expr_alias: [ ,  ] }
           LogicalFilter { predicate: (Field(Field($1, 1:Int32), 0:Int32) > 1:Int32) }
             LogicalScan { table: t, columns: [id, country, zipcode, rate, _row_id#0] }
+  create_source:
+    row_format: protobuf
+    name: s
+    file: |
+        syntax = "proto3";
+        package test;
+        message TestRecord {
+          int32 id = 1;
+          Country country = 3;
+          int64 zipcode = 4;
+          float rate = 5;
+        }
+        message Country {
+          string address = 1;
+          City city = 2;
+          string zipcode = 3;
+        }
+        message City {
+          int64 address = 1;
+          string zipcode = 2;
+        }
 - sql: |
     create table t (v1 bigint, v2 double precision);
     select (v1).v2 from t;
   binder_error: 'Item not found: Cannot get field from non nested column: v1'
 - sql: |
     create materialized view t as select * from s;
     select (country + country) from t;
+  binder_error: 'Feature is not yet implemented: Add[Struct, Struct], Tracking issue: https://github.com/singularity-data/risingwave/issues/112'
   create_source:
     row_format: protobuf
     name: s
     file: |
-      syntax = "proto3";
-      package test;
-      message TestRecord {
-        int32 id = 1;
-        Country country = 3;
-        int64 zipcode = 4;
-        float rate = 5;
-      }
-      message Country {
-        string address = 1;
-        string zipcode = 3;
-      }
-  binder_error: 'Feature is not yet implemented: Add[Struct, Struct], Tracking issue: https://github.com/singularity-data/risingwave/issues/112'
+        syntax = "proto3";
```

Comment:
What's going on with this file 🤣 My yq changed it throughly this morning, and yours seems to change again. Let's untracked this file for this PR?

Reply:
I don't know 🤣 I'll fix these